docs/data-ai/capture-data/advanced/how-sync-works.md

```diff
@@ -17,6 +17,7 @@ Data capture and cloud sync works differently for `viam-server` and `viam-micro-
 {{% tab name="viam-server" %}}
 
 The data is captured locally on the machine's storage and, by default, stored in the `~/.viam/capture` directory.
+For root (sudo) users, the `~/.viam/capture` directory resolves to `/root/.viam/capture`.
```

Comment:
This is only true for Linux. You can try this on a mac and it is instead `/var/root/.viam/`.

Reply:
```suggestion
For Linux root or sudo users, the `~/.viam/capture` directory resolves to `/root/.viam/capture`.
```
I think this is correct. Being a root user is not equivalent with running sudo. Worth getting reviewed by Nick again though.